wagtail_localize_rws_languagecloud/sync.py

```diff
@@ -358,6 +359,12 @@ def _import(client, logger):
                 db_project.internal_status = LanguageCloudProject.STATUS_IMPORTED
                 db_project.save()
 
+                translation_imported.send(
```

Comment:
You're probably closer to this project than I am. It is a while since I've been in any of the meetings but my understanding was that the requirement is they want the slack notification when _any_ translation is ready/imported rather than when _all_ of the translations for a project are imported, so I think this signal should really fire here

https://github.com/zerolab/wagtail-localize-rws-languagecloud/blob/ffe99fd3e6255e649b7407711d9a01b1e97c96ed/wagtail_localize_rws_languagecloud/sync.py#L334-L338

i.e: the same place where we currently send the email notification.

Reply:
Ah. Yes that makes sense. I will change this.

Reply:
Updated this.